tests/acceptance/suites/vertical-full-page-map.js

```diff
@@ -30,48 +33,44 @@ test('Clicking on a pin focuses on a results card', async t => {
 
 test('Search when map moves works', async t => {
   await SearchBar.submitQuery('virginia');
-  await SearchRequestLogger.waitOnSearchComplete();
-  const resultsCountBeforeDrag = await VerticalResults.getNumResults();
+  await SearchRequestLogger.waitOnSearchComplete(t);
   await ThemeMap.dragLeft();
   await ThemeMap.dragLeft();
-  await SearchRequestLogger.waitOnSearchComplete();
-  const resultsCountAfterDrag = await VerticalResults.getNumResults();
-  await t.expect(resultsCountBeforeDrag !== resultsCountAfterDrag).ok();
```

Comment:
do we no longer want this expect?

Reply:
the goal of this test is to see that dragging the map would trigger a search. Previously, we saved the results before and after to compare and check for a search. Now, we can just use the requestLogger to confirm that a search is fired!